src/Markup/Perspex.Markup/Data/PropertyAccessorNode.cs

```diff
@@ -50,11 +50,18 @@ protected override void SubscribeAndUpdate(WeakReference reference)
 
             if (instance != null && instance != PerspexProperty.UnsetValue)
             {
-                var plugin = ExpressionObserver.PropertyAccessors.FirstOrDefault(x => x.Match(reference));
+                var accessorPlugin = ExpressionObserver.PropertyAccessors.FirstOrDefault(x => x.Match(reference));
 
-                if (plugin != null)
+                if (accessorPlugin != null)
                 {
-                    _accessor = plugin.Start(reference, PropertyName, SetCurrentValue);
+                    _accessor = accessorPlugin.Start(reference, PropertyName, SetCurrentValue);
+                    foreach (var validationPlugin in ExpressionObserver.ValidationCheckers.Where(x => x.Match(reference)))
```

Comment:
Am I missing something or does this always add a validation plugin even if one's not requested?